aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -20,14 +24,17 @@ module aptos_std::ristretto255_bulletproofs {
     // Error codes
     //
 
-    /// There was an error deserializing the range proof.
-    const E_DESERIALIZE_RANGE_PROOF: u64 = 1;
```

Comment:
I don't understand: why are you removing this?

Reply:
Done